palvelutarjotin/settings.py

```diff
@@ -233,6 +237,36 @@
 CORS_ORIGIN_WHITELIST = env.list("CORS_ORIGIN_WHITELIST")
 CORS_ORIGIN_ALLOW_ALL = env.bool("CORS_ORIGIN_ALLOW_ALL")
 
+
+# Configure the default CSP rule for different source types
+CSP_DEFAULT_SRC = ["'self'"]
+
+# CSP_STYLE_SRC includes 'unsafe-inline' for inline styles added by `django-helusers`.
```

Comment:
The comment might be removed or not, I mean, it is in a way accurate because in another part of the code the unsafe-inline is added to CSP_STYLE_SRC when GraphiQL is enabled, but looking just here it's not evident.